test/hitl/card.binary/platformio.ini

```diff
@@ -0,0 +1,32 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:bw_swan_r5]
+
+[env]
+platform = ststm32
+board = bw_swan_r5
+framework = arduino
+build_flags =
+	-std=c++14
+	-Wall
+	-Werror # ensure unused functions in the test script generate an error
+	-D PIO_FRAMEWORK_ARDUINO_ENABLE_CDC
+	'-D PRODUCT_UID="com.blues.mat:test"'
```

Comment:
Once you iron out the bugs, get Brandon (or someone with special powers) to spin up a `com.blues.test` project for your tests.

Reply:
Yes. For now I'm playing in my own sandbox. This covers all the test runners and their associated assets such as Notehub projects. Longer term, I want to move all the runners to a Runners Group at organization level and will need some powers to be able to make that happen. We can then move all the notehub projects to a company account so they are no longer on my personal account.